### PR TITLE
virtuoso: add virtuoso-5 to support PowerPC

### PR DESCRIPTION
**Pull request by barracuda156**
Fixes: https://trac.macports.org/ticket/67840

#### Description

This will finally allow KDE4 stuff to build on PowerPC.

I skip CI, because `virtuoso-5` is set for `ppc ppc64`, and remaining changes are merely adding conflicts with it and its support into `virtuoso` (there is nothing to build there).
In the meanwhile, `virtuoso-6` is broken now across the board, so it will sabotage CI: https://ports.macports.org/port/virtuoso-6/details

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/virtuoso-5/Portfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+notes "
+Some antivirus software can cause the virtuoso build to hang indefinitely.\
+If you have any antivirus software installed, please disable it while\
+building virtuoso; it can be re-enabled after virtuoso has been installed.
+"
+
+livecheck.url       http://sourceforge.net/api/file/index/project-id/161622/mtime/desc/rss?path=/virtuoso
+livecheck.type      regex
+livecheck.regex     ${myname}/(6(\\.\\d+)+)/${myname}-opensource-(6(\\.\\d+)+)
```
Comment (nerdling) [inline devel/virtuoso-5/Portfile:67]: Looks like version 6 information for a version 5 portifle

Comment (nerdling) [reply devel/virtuoso-5/Portfile:67]: This is a non-blocker and can be fixed later.

**Comment by nerdling** [comment]
Looks like it should work; thanks for putting this together!

File: devel/virtuoso-5/Portfile (hunk 1 above)
Comment (barracuda156) [reply devel/virtuoso-5/Portfile:67]: I borrowed the existing port for v. 6 as a template and left this part as it were. Should we just drop this?

**Comment by barracuda156** [comment]
@nerdling Thank you very much for merging!

What is the issue with `virtuoso-6` by the way? If it is OpenSSL, then switching to using PG and setting `openssl.branch` to 1.1 or 1.0 might fix it.

**Pull request merged**